docs/contribute/release_process.rst

```diff
@@ -111,16 +117,16 @@ Create source code artifacts,
 
 .. code-block:: bash
 
-	git clone git@github.com:apache/tvm.git apache-tvm-src-v0.6.0.rc0
-	cd apache-tvm-src-v0.6.0.rc0
+	git clone git@github.com:apache/tvm.git apache-tvm-src-v0.6.0
```

Comment:
since folks are gonna replace the version number, maybe we should add a note to explain this change explicitly above?